src/components/Modal.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div
+    id="modal-fadeInScale-fs"
+    :class="{
+      'modal is-active': isActive,
+       'modal': !isActive,
+    }">
+    <div class="modal-background"></div>
+    <div class="modal-content">
+         <button
+      class="modal-button-close delete is-large"
+        aria-label="close"
+        v-on:click="handleModalClose()"
+      ></button>
+      <div class="has-text-centered">
+        <h2>{{ team.project.name }}</h2>
+        <p>{{ team.project.description }}</p>
+
+        <p class="socials">
+          <a :href="team.project.links.repository" target="_blank">
+           <i class="fab fa-github fa-2x"></i>
+          </a>
+          <a :href="team.project.links.website || placeholderDeployment" target="_blank">
```

Comment:
`website` and `writeup` are [optional](https://ubclaunchpad.github.io/new/config/globals.html#project) - we can get rid of the placeholders and just not render these divs using a `v-if="team.project.links.website"` directive

Reply:
Yep currently doing that

Reply:
I was thinking of also adding iOS and Android config options and icons. Currently in progress..

Reply:
> iOS and Android config options and icons

hold on to that thought, maybe open another ticket - let's keep this PR on-scope for the time being :) most notably I think it should wait until post- #46

Reply:
Yeah.. Which is why I removed the iconset stuff as well. I'll do it in another PR and make an issue for the config.

Reply:
Fixed in: c525c21